ur_robot_driver/doc/usage.rst

```diff
@@ -221,6 +223,7 @@ restarted again. Depending whether you use headless mode or not, you'll have to
 external_control program again.
 
 .. note::
+  Custom URScript command only works in headless mode. And does not work with the external_control URCap.
```

Comment:
I think, this  is  a bit misleading, as in fact it is working together with the external_control URCap. The program just has to be restarted afterwards, as stated above.

Reply:
@fmauch No that is not correct. On the e-series it only will work when the robot in remote control.
Maybe that is what we should write